modules/openapi-generator-gradle-plugin/src/main/kotlin/org/openapitools/generator/gradle/plugin/tasks/GenerateTask.kt

```diff
@@ -569,6 +574,10 @@ open class GenerateTask : DefaultTask() {
                 configurator.setInputSpec(value)
             }
 
+            remoteInputSpec.ifNotEmpty {value ->
```

Comment:
I feel we should [log a warning](https://docs.gradle.org/current/userguide/logging.html) if both `inputSpec` and `remoteInputSpec` are set. It should mention that `remoteInputSpec` takes priority over `inputSpec`.